src/gui/optionsdialog.ui

```diff
@@ -1103,6 +1103,16 @@ Manual: Various torrent properties (e.g. save path) must be assigned manually</s
                  </property>
                 </widget>
                </item>
+               <item>
+                <widget class="QCheckBox" name="checkUseCategoryPaths">
+                 <property name="text">
+                  <string>Use Category paths in "Manual" mode</string>
```

Comment:
oh... please don't invent another "word"... In options dialog `Manual Mode` is used without quotes, at least you should stick with it and stay coherent.

I would even go further and change all of it to lowercase (`manual mode`) as it isn't that important to capitalize it IMO.

Reply:
> oh... please don't invent another "word"... In options dialog `Manual Mode` is used without quotes, at least you should stick with it and stay coherent.

Ok.

Reply:
> I would even go further and change all of it to lowercase (`manual mode`) as it isn't that important to capitalize it IMO.

I feel it better if the names of options/values, etc. are emphasised in some way. In this case, it is clearly visible that we are talking about something specific, which should prevent misunderstandings.
In any case, if we decide to change this, it's better to do it in batches for the entire GUI.